packages/admin/cms-admin/src/preview/README.md

```diff
@@ -0,0 +1,83 @@
+Comet has two different previews:
```

Comment:
Nice 👍🏼  Maybe we can re-use parts of this documentation of https://docs.comet-dxp.com